store/sqlstore/user_store.go

```diff
@@ -667,12 +667,18 @@ func (us SqlUserStore) InvalidateProfilesInChannelCacheByUser(userId string) {}
 
 func (us SqlUserStore) InvalidateProfilesInChannelCache(channelId string) {}
 
-func (us SqlUserStore) GetProfilesInChannel(channelId string, offset int, limit int) ([]*model.User, *model.AppError) {
+func (us SqlUserStore) GetProfilesInChannel(options *model.UserGetOptions) ([]*model.User, *model.AppError) {
 	query := us.usersQuery.
 		Join("ChannelMembers cm ON ( cm.UserId = u.Id )").
-		Where("cm.ChannelId = ?", channelId).
+		Where("cm.ChannelId = ?", options.InChannelId).
 		OrderBy("u.Username ASC").
-		Offset(uint64(offset)).Limit(uint64(limit))
+		Offset(uint64(options.Page * options.PerPage)).Limit(uint64(options.PerPage))
+
+	if options.Inactive {
```

Comment:
0/5, but keeping in mind the user of the API, I wonder if we should filter to `options.Inactive && !options.Active` so that passing true for both does the "right thing" and returns all users. Similarly for `options.Active && !options.Inactive`. It's unrelated to the core changes here, of course.

Reply:
Makes sense to me. There are similar ways of doing this in the code though. I will update this to be safe side :)